.eslintignore

```diff
@@ -5,3 +5,4 @@ locale
 preset-react-app-custom
 build
 **/*.ts
+**/__tests__/**
```

Comment:
Once linting is Fixed™️, would be good to get tests covered by linting as well. But that's a not-now thing.